connectors/bitcoin.go

```diff
@@ -84,6 +87,22 @@ func (btc *BTC) Connect(endpoint string, username string, password string) error
 	if err != nil {
 		return fmt.Errorf("RPC client error: %v", err)
 	}
+
+	info, err := c.GetNetworkInfo()
+
+	switch err := err.(type) {
+	case nil:
+		log.Debugf("detected btcd version: %v", info.Version)
+	// Inspect the RPC error to ensure the method was not found, otherwise
+	// we actually ran into an error.
+	case *btcjson.RPCError:
+		if err.Code != btcjson.ErrRPCMethodNotFound.Code {
+			return fmt.Errorf("unable to detect btcd version: %v", err)
+		}
```

Comment:
Is it possible that the BTC node does not implement the getInfo method?

Reply:
new versions don't provide `getInfo`, it's deprecated. Instead there are `getnetworkinfo`, `getblockchaininfo` etc